### PR TITLE
Fixed animate line data parsing issue in FF

### LINKED ISSUE #2440: animate_line quicktest broken on firefox
Can't find any reason in the console.
I've run it a few times so it doesn't seem like a quicktest flake.
The line just isn't showing up.
None of the other line quicktests seem broken.
screen shot 2015-07-06 at 3 21 13 pm

### PR DESCRIPTION
**Pull request by aicioara**
Closes #2440
Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse

### CODE REVIEW

**Comment by aicioara** [comment]
Similar solution for https://github.com/palantir/plottable-website/issues/105

File: quicktests/overlaying/tests/animations/animate_line.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 
 function makeData() {
   "use strict";
-  var data = [{x: new Date("2014-04-01 00:24"), y: 4},
-              {x: new Date("2014-08-29 00:24"), y: 6}];
+  var data = [{x: new Date("4/1/2014 00:24"), y: 4},
+              {x: new Date("8/29/2014 00:24"), y: 6}];
```
Comment (rcchen) [inline quicktests/overlaying/tests/animations/animate_line.js]: Could we use the [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) standard instead? Just for the sake of eliminating confusion.

**Comment by rcchen** [comment]
Let's use the standard instead. I tested it in Firefox and new Date() will recognize it properly.

**Comment by aicioara** [comment]
This will break the test. We need a localized version of the date. 
```
> new Date("2014-04-01T00:24Z")
< Mon Mar 31 2014 17:24:00 GMT-0700 (PDT)
```

**Comment by crmorford** [comment]
wait, was the fix just in the test, not in Plottable? That's what it looks like in the diff.

**Comment by aicioara** [comment]
Yes, it's the Time fun. Firefox does not know how to parse dates like 
```typescript 
new Date("2014-04-01 00:24")
```
(if you open console and write that, you will get `Invalid Date`, while in Chrome it still works)

**Comment by crmorford** [comment]
ah okay, that's a relief :)

**Pull request merged**